docs/brew-tap.md

```diff
@@ -73,7 +73,7 @@ If you need a formula to be installed from a particular tap, you can use fully
 qualified names to refer to them.
 
 For example, you can create a tap for an alternative `vim` formula. Without
-pinning it, the behavior will be
```

Comment:
According to Prose-Style-Guidelines.md, Commonwealth English is preferred. This was the only instance in the docs of this spelling.